### ISSUE TITLE
Shopfront: displaying the name of the fee instead of the type

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/wishlist#126]
## What is the need / problem?

Today, we associate an entreprise fee to a domain, and 5 domains are available:

 - admin
  - sales
  - transport
  - packing
  - fundraising
  
  But these fee type are not easy to understand from the shopper pov. Instead of displaying the type in the pie chart, the idea here is to display the name.
  
  Maybe we want to limit the name to a specific amount of characters to avoid ugly shopfront piechart?

## Which type of users does this problem affect (and how many, if known)?

_- Shoppers_

## Connected wishlist and discovery discussions* [list precedent discussions]

https://community.openfoodnetwork.org/t/ux-enterprise-fees-and-price-breakdown/680

## Is Papercut or feature candidate?

Could be papercut

**Comment by jaycmb** [comment]
Updated description to make it a bit clearer what the task is.

So this papercut would only address changes to shopfront display then? Do we need another issue for invoices?

**Comment by RachL** [comment]
I thought it didn't fit to papercut to do both at the same time, but maybe I missunderstood?

**Comment by jaycmb** [comment]
Yep, I believe that´s what we said :)

I m just saying since in the original issue we also discussed invoices it should be consistant what the user sees as fees n the shopfront and in the end on their invoice?
I just imagine it could be confusing if we have different terms along the user journey (and counterproductive to the price transparency claim)

**Comment by RachL** [comment]
Yes a second issue is needed in that case, I was just checking if the two couldn't be done at the same time.
